app/javascript/viewer/uv_manager.js

```diff
@@ -137,6 +138,26 @@ export default class UVManager {
     }
   }
 
+  bindUvClick () {
+    const uvElement = document.getElementById('uv')
+    const uvClick = new Event("uv-click", { bubbles: true });
+
+    uvElement.addEventListener(
+      "click",
+      (event) => {
+        /* … */
+        uvElement.dispatchEvent(uvClick)
+        const cachedOpen = window.open
+        window.open = function (url, target, features) {
+          window.plausible('uv-click', { props: { url } })
```

Comment:
I don't think you need to override `window.open`, just put this below 149 and delete all this open stuff.

For the download controller that was more about attaching to the way UV does downloads.